index.html

```diff
@@ -169,20 +228,20 @@ <h1 class="about-me-title">About me</h1>
           </li>
         </ul>
       </form>
-      <ul class="nobull links">
-        <li>
-          <a class="social fa-brands fa-twitter" href="http://twitter.com/sps676"></a>
-        </li>
-        <li>
-          <a class="social fa-brands fa-linkedin" href="https://www.linkedin.com/in/sergiy-shkolnik-7801a53/"></a>
-        </li>
-        <li>
-          <a class="social fa-brands fa-github" href="https://github.com/SergSkol"></a>
-        </li>
-      </ul>
-      <section class="email">
-        <p>sps676@gmail.com</p>
-      </section>
+    </section>
+    <ul class="nobull links">
+      <li>
+        <a class="social fa-brands fa-twitter" href="http://twitter.com/sps676"></a>
+      </li>
+      <li>
+        <a class="social fa-brands fa-linkedin" href="https://www.linkedin.com/in/sergiy-shkolnik-7801a53/"></a>
+      </li>
+      <li>
+        <a class="social fa-brands fa-github" href="https://github.com/SergSkol"></a>
+      </li>
+    </ul>
+    <section class="email">
+      <p>sps676@gmail.com</p>
```

Comment:
- [ ] [OPTIONAL]  I see that there are some missing illustration design please make sure to add them to make your portfolio beautiful and also match the Figma design 
- Yours
<img width="1440" alt="Screen Shot 2022-10-05 at 15 41 12" src="https://user-images.githubusercontent.com/61790587/194088942-420a3273-73b8-4c1c-96d4-b120f716406d.png">
- Figma
<img width="813" alt="Screen Shot 2022-10-05 at 15 41 26" src="https://user-images.githubusercontent.com/61790587/194088976-4ae21874-ad70-4fb2-a2d9-9bedf74a301d.png">

Reply:
Thank you! Done!